video/out/cocoa/video_view.m

```diff
@@ -28,6 +28,9 @@ - (id)initWithFrame:(NSRect)frame
     if (self) {
         [self setAutoresizingMask:NSViewWidthSizable|NSViewHeightSizable];
     }
+    // Experimental #HDR on #macOS        
+    // This must be enabled to be able to display HDR content
+    self.wantsExtendedDynamicRangeOpenGLSurface = true;
```

Comment:
you can scrap the changes in this file. it's the old deprecated backend, that is also broken.

Reply:
Cool!